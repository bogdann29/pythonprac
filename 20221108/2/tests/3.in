(1,1), (1,1), (11,11)
sdfh
(1, 2), (3, 4)
1, 2, 3
(a, b), (c, d), (e, f)
(1,2), (4,5), (9,9)
