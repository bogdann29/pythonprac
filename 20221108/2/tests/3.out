Not a triangle
InvalidInput
InvalidInput
InvalidInput
InvalidInput
1.5
